operator/templates/cassandra-env-sh.yaml

```diff
@@ -247,34 +247,32 @@ data:
     # JMX connections.
     # For security reasons, you should not expose this port to the internet.  Firewall it if needed.
     JMX_PORT="{{ .Params.JMX_PORT }}"
+    RMI_PORT="{{ .Params.RMI_PORT }}"
 
-    {{ if eq .Params.JMX_LOCAL_ONLY "true" }}
-      JVM_OPTS="$JVM_OPTS -Dcassandra.jmx.local.port=$JMX_PORT"
-      JVM_OPTS="$JVM_OPTS -Dcom.sun.management.jmxremote.authenticate=false"
-    {{ else }}
+    {{ if ne $.Params.JMX_LOCAL_ONLY "true" }}
```

Comment:
Did you mean `if eq $.Params.JMX_LOCAL_ONLY "false"`? @nfnt

Reply:
That, or re-arranging this if-clause to handle the `eq == "true"` case first. While that will work here, it'll result in inconsistencies, because there are other if-clauses for `JMX_LOCAL_ONLY` that are _only_ for `JMX_LOCAL_ONLY == false`. Thus, I'm okay with keeping this as it is right now, because it'll be consistent.

Reply:
I'll resolve this, nothing to see here 😆